README.md

```diff
@@ -1,37 +1,28 @@
 # Zapier Apify Integration
-<a href="https://travis-ci.com/apifytech/apify-zapier-integration?branch=master"><img src="https://travis-ci.com/apifytech/apify-zapier-integration.svg?branch=master" alt="Build Status" style="display:inherit;"></a>
 
 The Apify integration to Zapier allows you to connect Apify platform with 600+ app.
 
-## How it works
+## Resources
 
-You can follow [Apify integration](https://zapier.com/apps/Apify/integrations) page on Zapier platform.
+* [Apify documentation](https://docs.apify.com/tutorials/integrations#get-started)
+* [Getting started tutorial](https://help.apify.com/en/articles/3034235-getting-started-with-apify-integration-for-zapier)
+* [Apify integration](https://zapier.com/apps/Apify/integrations) page on Zapier platform
 
 
 ## Development
 
+If you are interested in adding a new feature or fix a bug in integration, feel free to open PR.
```

Comment:
`or fix a bug in integration, feel free to open PR.` => `or fixing a bug in the integration, feel free to open a pull request.`